docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/paragraph--cgov-card-internal.html.twig

```diff
@@ -20,6 +26,9 @@
     {% set description = content.field_override_card_description[0] %}
   {% elseif content.field_featured_item[0]["#node"].field_feature_card_description[0] %}
```

Comment:
Now that the templates are combined, the description checks need to also check the card type. This current approach makes it too easy for a developer to accidentally include extra fields in their Display views and then introduce a hard to find bug.

Reply:
Updated logic to use cardType variable rather than just picking fields available to those card types